Apps/Admin/Tests/Functional/cypress/integration/e2e/delegation.cy.js

```diff
@@ -93,3 +94,131 @@ describe("Delegation", () => {
         cy.get("[data-testid=delegate-table]").should("not.exist");
     });
 });
+
+describe("Delegation Protect", () => {
+    beforeEach(() => {
+        cy.login(
+            Cypress.env("keycloak_username"),
+            Cypress.env("keycloak_password"),
+            "/delegation"
+        );
+    });
+
+    it("Verify protect dependent toggle, save button, cancel confirmation button and cancel button are selected.", () => {
+        cy.get("[data-testid=query-input]")
+            .clear()
+            .type(dependentWithGuardian.phn);
+        cy.get("[data-testid=search-button]").click();
+
+        // Protect dependent toggle
+        cy.get("[data-testid=dependent-protected-switch]").should(
+            "have.attr",
+            "aria-checked",
+            "false"
+        );
```

Comment:
Can we use the same style of assertions for the switch as the delegation search tests above?

```
cy.get("[data-testid=dependent-protected-switch]").should(
    "not.be.checked"
);
```